appveyor.yml

```diff
@@ -1,40 +1,30 @@
-# version here is only relevant for the AppVeyor CI, no need to adjust
-version: '{build}'
+version: '0.4.1.{build}'
 
 environment:
-  global:
-    PYTHON_LOC: "C:\\Miniconda36-x64"
+#  global:
+      # Example can be found here https://github.com/quantopian/zipline/blob/master/appveyor.yml
+#      ANACONDA_TOKEN: # not required anymore since it is configured at AppVeyor
```

Comment:
Why don't we need this?
Otherwise delete outcommented lines or add another comment why it should be kept.

Reply:
Can someone tell me how to activate appveya for this repo?